jellyfin_apiclient_python/api.py

```diff
@@ -125,10 +124,12 @@ def artwork(self, item_id, art, max_width, ext="jpg", index=None):
 
         return self._get_url(handler, params)
 
-    def audio_url(self, item_id, max_streaming_bitrate=140000000):
+    def audio_url(self, item_id, container, audio_codec=None, max_streaming_bitrate=140000000):
```

Comment:
Maybe allow container to default to None and don't pass it if it isn't set? This would preserve backward compatibility and make the API possibly easier to use.

Reply:
Yes, good idea. Done.